applications/nrf5340_audio/src/bluetooth/Kconfig

```
@@ -378,11 +436,15 @@ config AURACAST
 
 menu "QoS"
 
-config BT_AUDIO_PRESENTATION_DELAY_US
-	int "Presentation delay"
+config BT_AUDIO_LOCAL_PRESENTATION_DELAY_US
```

```suggestion
config BT_AUDIO_PRESENTATION_DELAY_US
```
I'm open for discussing this, but I think we should keep this "clean". This is located in the QoS section of the menuconfig so it should be clear that this is only set by the source/client. And local sounds like it is only used on this one device.